catalyst-gateway/bin/src/service/api/cardano/staking/assets_get.rs

```diff
@@ -79,10 +80,9 @@ struct TxoAssetInfo {
     /// Asset hash.
     id: Vec<u8>,
     /// Asset name.
-    // TODO: https://github.com/input-output-hk/catalyst-voices/issues/1121
-    name: String,
+    name: AssetName,
     /// Asset amount.
-    amount: num_bigint::BigInt,
+    amount: i128,
```

Comment:
```suggestion
    amount: num_bigint::BigInt,
```
It seems unnecessary to convert this to i128.
We already have a conversion that can convert BigInt to the required response type.
Otherwise we just do double conversion and validity checking for no gain.